BraintreeDropIn.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,25 +1,23 @@
 {
-  "object": {
-    "pins": [
-      {
-        "package": "Braintree",
-        "repositoryURL": "https://github.com/braintree/braintree_ios",
-        "state": {
-          "branch": null,
-          "revision": "3924f5ea0295087081e7e637fbf1333361e7054b",
-          "version": "5.23.0"
-        }
-      },
-      {
-        "package": "InAppSettingsKit",
-        "repositoryURL": "https://github.com/futuretap/InAppSettingsKit.git",
-        "state": {
-          "branch": null,
-          "revision": "71e03ebad8462618f96c575c86d2e9220f880676",
-          "version": "3.3.2"
-        }
+  "pins" : [
+    {
+      "identity" : "braintree_ios",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/braintree/braintree_ios",
+      "state" : {
+        "revision" : "619e78a9ab9423be4871baf406546c22573086cf",
+        "version" : "5.25.0"
       }
-    ]
-  },
-  "version": 1
+    },
+    {
+      "identity" : "inappsettingskit",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/futuretap/InAppSettingsKit.git",
+      "state" : {
+        "revision" : "71e03ebad8462618f96c575c86d2e9220f880676",
+        "version" : "3.3.2"
+      }
+    }
+  ],
+  "version" : 2
```

Comment:
Can we do some digging and just make sure there aren't any issues moving this to version 2?

Reply:
Yes, that's what I am looking up. They are also missing the "object" key that was there before. 
So I think something is wrong.

Reply:
I found this post: https://forums.swift.org/t/using-swiftpm-5-6-and-supporting-older-versions/55708

Reply:
It is recommended by a few people here is to add Package.resolved to .gitignore file

Reply:
We want to leave it so that we are all using the same version as far as adding it to the `.gitignore`. Lets run a test on 15.0 which is our minimally supported Xcode version in the SDK. If that looks good we can apply this change as is.

Reply:
SPMTest looks good with Xcode 15.0 pointing to this branch.

Reply:
Thanks for triple checking! ❤️